src/scripts/utils/salaryParser.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const salaryParser = (salary) => {
```

Comment:
it's not good idea to create parser that can be able to format from number to correct string and from correct string to number. Create function with only 1 task